mappings/mp_efo_impc.sssom.tsv

```diff
@@ -0,0 +1,100 @@
+MP:0000181	abnormal circulating LDL cholesterol level	skos:broadMatch	EFO:0004611	low density lipoprotein cholesterol measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000184	abnormal circulating HDL cholesterol level	skos:relatedMatch	EFO:0008592	high density lipoprotein particle size measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000188	abnormal circulating glucose level	skos:broadMatch	EFO:0004468	glucose measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000199	abnormal circulating serum albumin level	skos:broadMatch	EFO:0004535	serum albumin measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000202	abnormal circulating alkaline phosphatase level	skos:broadMatch	EFO:0004533	alkaline phosphatase measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000203	abnormal circulating aspartate transaminase level	skos:broadMatch	EFO:0004736	aspartate aminotransferase measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0000226	abnormal mean corpuscular volume	skos:broadMatch	EFO:0004526	mean corpuscular volume	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001259	abnormal body weight	skos:broadMatch	EFO:0004338	body weight	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001553	abnormal circulating free fatty acids level	skos:broadMatch	EFO:0005110	fatty acid measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001560	abnormal circulating insulin level	skos:broadMatch	EFO:0004467	insulin measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001562	abnormal circulating calcium level	skos:broadMatch	EFO:0004838	calcium measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001565	abnormal circulating phosphate level	skos:broadMatch	EFO:0010968	phosphate measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001565	abnormal circulating phosphate level	skos:broadMatch	EFO:0010972	blood phosphate measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001569	abnormal circulating bilirubin level	skos:broadMatch	EFO:0004570	bilirubin measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001573	abnormal circulating alanine transaminase level	skos:broadMatch	EFO:0004735	serum alanine aminotransferase measurement	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140
+MP:0001575	cyanosis	skos:closeMatch	EFO:0009447	hypoxemia	semapv:ManualMappingCuration	MP	EFO	1	2024-08-16	orcid:0000-0001-8314-2140

```

Comment:
I'm not sure why this is called close, cyanosis is the skin discoloration that results from low blood oxygen. If the point is to get the best match for the EFO term hypoxemia then why not use the MP term hypoxemia (MP:0012549), which would be an exact match. 
Another question, why is there a phenotype term in EFO? Wouldn't the experimental factor be low oxygen.

Reply:
The targeted set of MP terms are those that have been requested by IMPC to be mapped to EFO so that their data can related to GWAS Catalog genetic associations. I suspect that these are the terms with existing associated IMPC data. It seems they have no [hypoxemia ](https://www.mousephenotype.org/data/phenotypes/MP:0005288) associated annotations.

>"why is there a phenotype term in EFO?" 

There are all kinds of terms in EFO. I guess the phenotype terms have been added in response to requests from phenotype curators associated with EBI.